aiida_quantumespresso/calculations/projwfc.py

```diff
@@ -61,10 +61,6 @@ def define(cls, spec):
             message='The retrieved temporary folder could not be accessed.')
         spec.exit_code(303, 'ERROR_OUTPUT_XML_MISSING',
             message='The retrieved folder did not contain the required XML file.')
-        spec.exit_code(310, 'ERROR_OUTPUT_STDOUT_READ',
-            message='The stdout output file could not be read.')
-        spec.exit_code(312, 'ERROR_OUTPUT_STDOUT_INCOMPLETE',
-            message='The stdout output file was incomplete probably because the calculation got interrupted.')
         spec.exit_code(320, 'ERROR_OUTPUT_XML_READ',
```

Comment:
Think these XML exit codes are also kind of generic for codes that have an XML output. Shouldn't `PwCalculation` also have these exit codes? If so might be worth looking into abstracting them as well.

Reply:
I agree entirely! But I will leave refactoring the XML parsing to another PR for v4.0, where we can also clean up that part of the code significantly when we no longer support older versions of QE. I've added a note to https://github.com/aiidateam/aiida-quantumespresso/issues/748 on this.